tsdb/compact.go

```diff
@@ -220,6 +220,14 @@ func (c *LeveledCompactor) plan(dms []dirMeta) ([]string, error) {
 			return []string{dms[i].dir}, nil
 		}
 	}
+	// If the block is entirely deleted, then we don't care about
```

Comment:
Let's add comment on line 291 `TODO: This is assuming single tombstone is for distinct series, which might be no true`